core/src/main/java/dev/streamx/cli/run/RunCommand.java

```diff
@@ -67,6 +77,8 @@ public void run() {
       Quarkus.waitForExit();
     } catch (IOException e) {
       throw new RuntimeException("Cannot run StreamX", e);
+    } catch (ContainerLaunchException e) {
+      throw DockerException.containerStartupFailed(e.getMessage());
```

Comment:
Do we want to throw an exception here? It's known and "expected".
we should check if it's a timeout and return prompt.